README.md

```diff
@@ -1,12 +1,12 @@
 # Landscape Documentation
 
-Landscape is a systems management tool designed to help you manage and monitor your Ubuntu systems from a unified platform.
+Landscape is a systems management tool for managing and monitoring Ubuntu deployments, ranging from a few machines to large environments. It lets you control configuration, updates, and security from a central location instead of handling each machine separately.
```

Comment:
It lets you push security and bugfix updates, as well as configure, harden, audit, and inventory software and users.

Reply:
I've removed the part you suggested about handling machines separately, but I kept the earlier parts only because your suggested wording is very similar to what's in the second paragraph, so it didn't read well when all put together. The content of what you've suggested is still in the doc though